recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -82,54 +70,83 @@ def requirements(self):
         self.requires("fft/cci.20061228")
         self.requires("flatbuffers/2.0.5")
         self.requires("gemmlowp/cci.20210928")
+        self.requires("opencl-headers/2022.05.18")
+        self.requires("vulkan-headers/1.3.221")
         if self.settings.arch in ("x86", "x86_64"):
             self.requires("intel-neon2sse/cci.20210225")
-        self.requires("ruy/cci.20220628")
+        if self.options.with_ruy or True: # Upstream always calls find_package
```

Comment:
```suggestion
        if self.options.with_ruy or True: # FIXME: Upstream always calls find_package
```

Maybe someone provides a patch for the build-system